open_academy/views/course_views.xml

```diff
@@ -29,13 +29,19 @@
             <search>
                 <field name="title"/>
                 <field name="description"/>
+                <field name="responsible_user_id"/>
+                <filter name="current_user" string="Current user" domain="[('responsible_user_id', '=', uid)]"/>
+                <group string="Group By">
+                    <filter name="group_by_responsible_users" string="Responsible users" context="{'group_by': 'responsible_user_id'}"/>
+                </group>
             </search>
         </field>
     </record>
     <record id="course_action" model="ir.actions.act_window">
         <field name="name">Courses</field>
         <field name="res_model">course</field>
         <field name="view_mode">tree,form</field>
+        <field name="context">{'search_default_current_user': 1}</field>
```

Comment:
Why using the number 1?

Reply:
Because with number 1 I set this filter as the default filter, otherwise it will not appear by default

Reply:
By the name `search_default_current_user` I thought that the number '1' is the ID of a user, are you sure about this?

Reply:
Yes, I am sure, when I put a 0 the filter is disabled automatically, and the current_user is just the name of the filter

Reply:
I thought it has something to do with users or partners, but I forgot that the filter name is current_user